templates/rhpam74-authoring.yaml

```diff
@@ -836,7 +836,7 @@ objects:
             readOnly: true
 ## H2 volume mount BEGIN
           - name: "${APPLICATION_NAME}-h2-pvol"
-            mountPath: "/opt/eap/standalone/data"
+            mountPath: "/opt/kie/data"
```

Comment:
If this mountPath is going to be "/opt/kie/data" instead of "/opt/kie/data/h2", then perhaps shouldn't "h2-pvol" get renamed to "kie-pvol", and "h2-claim" get renamed to "kie-claim"?

Reply:
@errantepiphany this will be handled in https://issues.jboss.org/browse/KIECLOUD-217. Don't you rather wait for it?

Reply:
@errantepiphany this will be handled in https://issues.jboss.org/browse/KIECLOUD-217. Don't you rather wait for it?

Reply:
That works.